tests/phpunit/tests/wp-token-map/wpTokenMap.php

```diff
@@ -0,0 +1,400 @@
+<?php
+/**
+ * Unit tests covering WP_Token_Map functionality.
+ *
+ * @package WordPress
+ *
+ * @since 6.6.0
+ *
+ * @coversDefaultClass WP_Token_Map
+ */
+class Tests_WpTokenMap extends WP_UnitTestCase {
+	/**
+	 * Number of unique HTML5 named character references, including
+	 * variations of a given name that don't require the trailing semicolon.
+	 *
+	 * The set of names is fixed by the specification,
+	 * and can be found at the following link.
+	 *
+	 * @link https://html.spec.whatwg.org/entities.json
+	 */
+	const KNOWN_COUNT_OF_ALL_HTML5_NAMED_CHARACTER_REFERENCES = 2231;
+
+	/**
+	 * Small test array matching names to Emoji.
+	 *
+	 * @var array.
+	 */
+	const ANIMAL_EMOJI = array(
+		'cat'     => '🐈',
+		'dog'     => '🐶',
+		'fish'    => '🐟',
+		'mammoth' => '🦣',
+		'seal'    => '🦭',
+	);
+
+	/**
+	 * Returns an associative array whose keys are tokens to replace and
+	 * whose values are the replacement strings for those tokens.
+	 *
+	 * This function is here to help avoid bloating this specific test file.
+	 * For example, the HTML5 dataset is very large and best served as a
+	 * separate file.
+	 *
+	 * The HTML5 named character reference list is pulled directly from the
+	 * WHATWG spec and stored in the tests directory so it doesn't need to
+	 * be downloaded on every test run. By specification, it cannot change
+	 * and will not be updated.
+	 *
+	 * @ticket 60698
```

Comment:
As this is a helper rather than a test, the ticket number isn't necessary

Reply:
removed in 0e7ab4ca52d97da25a20e0e493bc33308f56cf1d